src/com/dre/brewery/recipe/RecipeItem.java

```diff
@@ -76,6 +76,11 @@ public abstract class RecipeItem implements Cloneable {
 	@Nullable
 	public abstract List<Material> getMaterials();
 
+	/**
+	 * @return A user-displayable name for this recipeItem
+	 */
+	public abstract String displayName();
```

Comment:
I think a default implementation for this should be provided in the `PluginItem`. Other Plugins that provide their own PluginItem already would otherwise fail now